src/sentry/sentry_metrics/querying/data_v2/transformation/base.py

```diff
@@ -6,7 +6,7 @@
 QueryTransformerResult = TypeVar("QueryTransformerResult")
 
 
-class QueryTransformer(ABC, Generic[QueryTransformerResult]):
+class QueryResultsTransformer(ABC, Generic[QueryTransformerResult]):
```

Comment:
This is a transformer of the results and not the query, so I have changed the name.